client/components/Condition/Condition.js

```diff
@@ -10,16 +11,32 @@ export default class Condition extends PureComponent {
   };
 
   handleOnBlurAction = (e) => {
-    this.props.action(e.target.value);
+    const { condition: { price } } = this.props;
```

Comment:
price has nothing to do with updating the radius field. why do we need to setPrice everytime we change the radius?

Reply:
did some changes here. please check thanks